data/json/terrain.json

```diff
@@ -119,6 +124,7 @@
         "type" : "terrain",
         "id" : "t_sconc_wall_halfway",
         "name": "half-built simple concrete wall",
+        "description": "A partially poured concrete wall that could probably keep anything behind it quite safe if it were finished.",
```

Comment:
Recommend switching to `would keep anything` to make this a bit more forceful and less passive